core/lib/generators/spree/dummy/templates/rails/test.rb

```diff
@@ -20,6 +20,9 @@
   # Raise exceptions instead of rendering exception templates
   config.action_dispatch.show_exceptions = false
 
+  # Disable request forgery protection in test environment
+  config.action_controller.allow_forgery_protection = false
```

Comment:
I feel like this is not necessary since we already set this value when defining the dummy app.

Reply:
That's a good point. It is redundant, I just pushed a change removing the line. Thanks

Reply:
Hey!
I have tried to run the [solidus_starter_frontend](https://github.com/nebulab/solidus_starter_frontend/pull/90) from this branch and they are failing because of the forgery protection.

I think we need to disable the `allow_forgery_protection` configuration here as well.

The reason is that there are two dummy apps that can be generated.

 1 - A `Dummy::Application` to test extensions that is created by running [this](https://github.com/solidusio/solidus/blob/v2.11.4/core/lib/spree/testing_support/extension_rake.rb) task and use this [DummyGenerator](https://github.com/solidusio/solidus/blob/v2.11.4/core/lib/generators/spree/dummy/dummy_generator.rb#L61). This generator creates a full rails application (Dummy::Application) and overrides the rails `environments/test.rb` file with the above templates/rails/test.rb configuration. This is the reason `allow_forgery_protection` needs to be disabled in `core/lib/generators/spree/dummy/templates/rails/test.rb `.
 
2 - A `DummyApp::Application` to test the solidus framework and it is an application initialized in-memory [here](https://github.com/solidusio/solidus/blob/v2.11.4/core/lib/spree/testing_support/dummy_app.rb#L35) when the [spec/rails_helper.rb](https://github.com/solidusio/solidus/blob/master/core/spec/rails_helper.rb#L8) file is loaded. This is the reason we need to disable in `core/lib/spree/testing_support/dummy_app.rb`

Reply:
You are right @SamuelMartini, we need to re-add it to make it work with Dummy apps generated in extensions. 

@FrancescoAiello01 sorry for the wrong suggestion here! 😬

Reply:
That makes sense, sorry for the confusion! Going to push a fix now.